server/methods.js

```diff
@@ -88,16 +88,17 @@ Meteor.startup(function() {
       }
     },
     deleteNotification: function(notification){
+      check(notification._id, String);
```

Comment:
This works, but a nicer syntax is to pass and object and properties to check as an object:

```
check(notification, {
  _id: String,
  initiativeId: String,
  userId: String,
  type: String,
  isRead: Boolean,
  createdAt: Number
});
```